esx_service/cli/vmdkops_admin.py

```diff
@@ -922,7 +922,10 @@ def tenant_vm_ls(args):
     rows = generate_tenant_vm_ls_rows(vms)
     print(cli_table.create(header, rows))
 
-
+# Allowing upper/lower case true, false values but not
+# any random strings
+def validate_create_value(allow_create):
```

Comment:
Suggest to rename this method to validate_allow_create or validate_allow_create_value.